Source/Scene/Expression.js

```diff
@@ -541,27 +541,39 @@ define([
 
     function parseKeywordsAndVariables(ast) {
         if (isVariable(ast.name)) {
-            return new Node(ExpressionNodeType.VARIABLE, getPropertyName(ast.name));
+            var name = getPropertyName(ast.name);
+            if (name.substr(0, 8) === 'tiles3d_') {
```

Comment:
In the spec, can you document that variable names starting with `tiles3d_` are served for built-ins?